feature/staff/src/main/kotlin/io/github/droidkaigi/confsched2023/staff/section/StaffSheet.kt

```diff
@@ -0,0 +1,70 @@
+package io.github.droidkaigi.confsched2023.staff.section
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import io.github.droidkaigi.confsched2023.model.Staff
+import io.github.droidkaigi.confsched2023.staff.section.StaffSheetUiState.Loading
+import io.github.droidkaigi.confsched2023.staff.section.StaffSheetUiState.StaffList
+import kotlinx.collections.immutable.PersistentList
+
+internal sealed interface StaffSheetUiState {
```

Comment:
ditto. Custom interfaces must be annotated with Stable to avoid recomposition by unexpectedly breaking stability.

```suggestion
@Stable
internal sealed interface StaffSheetUiState {
```

Reply:
fixed [c60a497](https://github.com/DroidKaigi/conference-app-2023/pull/730/commits/c60a497294b04974eb92dd1b94d6da3045686221)